src/components/file-dropzone/examples/file-dropzone-type-filtering.tsx

```diff
@@ -0,0 +1,72 @@
+import { FileInfo } from '@limetech/lime-elements';
+import { Component, h, State } from '@stencil/core';
+
+/**
+ * File type filtering
+ * The component allows you to specify the types of files that the dropzone will accept.
+ * By default, it accepts all file types (`*`).
+ *
+ * For media files, it is possible to specify any format, using:
+ * `audio/*`, `video/*`, `image/*`.
+ *
+ * Additionally, you can use unique file type specifiers, such as:
+ * `.jpg`, or `.pdf`; or use a comma-separated list of file extensions or MIME types,
+ * for instance: `image/png, image/jpeg` or `.png, .jpg, .jpeg`.
+ *
+ * Read more about
+ * [HTML attribute: accept](https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes/accept)
+ */
+@Component({
+    tag: 'limel-example-file-dropzone-type-filtering',
+    shadow: true,
+    styleUrl: 'file-dropzone-type-filtering.scss',
+})
+export class FileDropzoneTypeFilteringExample {
+    @State()
+    private files: FileInfo[] = [];
+
+    @State()
+    private rejectedFiles: FileInfo[] = [];
+
+    public render() {
+        return [
+            <limel-file-dropzone
+                onFilesSelected={this.handleAcceptedFiles}
+                onFilesRejected={this.handleRejectedFiles}
+                accept="image/*"
+                text="Drop pictures here"
+            >
+                <limel-input-field
+                    label="Dream pet description"
+                    type="textarea"
+                    placeholder="What is your dream pet? Describe it here and attach a picture of it."
+                    helperText="This text will be displayed in your profile"
```

Comment:
This is a good example where the user will not even dream about dragging and dropping a file for uploading it.

<img width="449" alt="image" src="https://github.com/Lundalogik/lime-elements/assets/35954987/b56664d1-0bf3-4d91-b756-3d6331ca27c5">

For now, I will add instructions to the example itself. But I think we should also add this note to the docs.